cg/services/order_validation_service/workflows/tomte/validation_rules.py

```diff
@@ -18,3 +19,5 @@
 ]
 
 TOMTE_CASE_SAMPLE_RULES = [validate_wells_contain_at_most_one_sample]
+
+TOMTE_SAMPLE_RULES = [validate_application_exists]
```

Comment:
This should be case sample rule. The sample rules are for samples which can be uniquely identified by the sample name alone, so the microbial stuff.

All tomte samples require both a case name and a sample name to be uniquely identified in the order.

Reply:
Okay, that naming is very confusing. I interpreted CASE_SAMPLE_RULES as rules which are applied to the samples within a given case. Could we clear these namings up?

Reply:
That is the correct interpretation 👍

Reply:
Resolved in person